lib/extend.js

```diff
@@ -81,10 +81,10 @@ var combineJSON = require('./utils/combineJSON'),
 function extend(opts) {
   var options, to_ret;
 
-  // Overloaded method, can accept a string as a path that points to a JSON file
-  // or a plain object. Potentially refactor.
+  // Overloaded method, can accept a string as a path that points to a JS or
+  // JSON file or a plain object. Potentially refactor.
   if (_.isString(opts)) {
-    options = fs.readJsonSync(opts);
+    options = require(path.resolve(process.cwd(), opts));
```

Comment:
Just a conversation starter, the recent addition to do something similar for style dictionary files (#89) used the resolve-cwd module. Open question to you and @jreichenberg: should we use resolve-cwd here as well? Should we remove the resolve-cwd module and use this way in combineJSON.js? Does it matter?

Reply:
I missed that dependency. Not sure why it's necessary, but I don't really care one way or the other happy to add it in for consistency. @jreichenberg what's the value add of that dependency?

Reply:
@elliotdickison See the discussion in 89: https://github.com/amzn/style-dictionary/pull/89#discussion_r189397621

Both the native node require() statement and readJsonSync will parse any JSON you throw at it. The difference is that native require() will resolve packages in the process (yay!) but needs some help resolving a path relative to the cwd (boo!). So kind of depends on the desired capability.  I know that doesn't help much...